example/migrations/0002_auto_20160513_0857.py

```diff
@@ -9,7 +9,6 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('contenttypes', '0002_remove_content_type_name'),
```

Comment:
It breaks on Django 1.7 since migration `0002` does not exist (added in 1.8). But for the purpose of this migration, the initial content type migration seems fine as a dependency.